Classes/Projection/ProjectionManager.php

```diff
@@ -88,18 +88,29 @@ public function getProjection(string $projectionIdentifier): Projection
     }
 
     /**
+     * Replay events of the specified projection
+     *
      * @param string $projectionIdentifier
-     * @return void
+     * @return integer Number of events which have been replayed
      */
     public function replay(string $projectionIdentifier)
     {
+        $eventCount = 0;
         $projection = $this->getProjection($projectionIdentifier);
+
+        $projector = $this->objectManager->get($projection->getProjectorClassName());
+        if ($projector instanceof ProjectorInterface) {
```

Comment:
that's actually not necessary, because the object manager will first complain if the object doesn't exist. And if the returned object is not a projector … well, then the user will probably will see it equally well with the PHP error message.